sql.js

```diff
@@ -61,7 +61,7 @@ const applySelect = ({ select, payload, context }) => {
     const { alias, field } = part
     const key = alias || field
     if (field === '*') {
-      event[key] = json
+      alias ? event[key] = json : Object.assign(event, json);
```

Comment:
@MikeTheReader sorry to nitpick, but could you make this an if/else?

Reply:
e.g.
```js
if (alias) event[key] = json
else Object.assign(event, json)
```

Reply:
Done (though I included the brackets).